test/ILP/default-ilp-client-test.ts

```diff
@@ -63,17 +56,72 @@ describe('Default ILP Client', function(): void {
     })
   })
 
+  it('Get balance - Not Found Error', function(done): void {
+    // GIVEN a DefaultIlpClient with a network client which always throws a NOT_FOUND error
+    const client = FakeDefaultIlpClients.fakeNotFoundErrorClient()
+
+    // WHEN the balance for an account is requested
+    client.getBalance('test.foo.bar').catch((error) => {
+      // THEN the error is translated to a XpringIlpError
+      assert.equal(error as XpringIlpError, XpringIlpError.accountNotFound)
+      done()
+    })
+  })
+
+  it('Get balance - Invalid Argument Error', function(done): void {
+    // GIVEN a DefaultIlpClient with a network client which always throws an INVALID_ARGUMENT error
+    const client = FakeDefaultIlpClients.fakeInvalidArgumentErrorClient()
+
+    // WHEN the balance for an account is requested
+    client.getBalance('test.foo.bar').catch((error) => {
+      // THEN the error is translated to a XpringIlpError
+      assert.equal(error as XpringIlpError, XpringIlpError.invalidArgument)
+      done()
+    })
+  })
+
+  it('Get balance - Unauthenticated Error', function(done): void {
+    // GIVEN a DefaultIlpClient with a network client which always throws an UNAUTHENTICATED error
+    const client = FakeDefaultIlpClients.fakeUnauthenticatedErrorClient()
+
+    // WHEN the balance for an account is requested
+    client.getBalance('test.foo.bar').catch((error) => {
+      // THEN the error is translated to a XpringIlpError
+      assert.equal(error as XpringIlpError, XpringIlpError.unauthenticated)
+      done()
+    })
+  })
+
+  it('Get balance - Internal Error', function(done): void {
+    // GIVEN a DefaultIlpClient with a network client which always throws an INTERNAL error
```

Comment:
I was confused at first about why these GIVEN statements reference all-caps errors such as `INTERNAL` whereas the last one references a `XpringIlpError.invalidAccessToken`.  Then I realized (I think) that the all-caps error types are actually gRPC errors, whereas the `invalidAccessToken` is originally thrown as a `XpringIlpError` .... (I think).  It might be worth qualifying these all-caps error names to make it obvious that they are native gRPC errors, like you do in the fake-default-ilp-client